index.html

```diff
@@ -147,40 +169,58 @@ <h2>Uploading New Materials</h2>
 				<p style="padding-bottom: 20px;">Your files should automatically upload. There is no limit to the file sizes you can upload however, please be aware that the larger the file, the longer it will take to upload. Depending on how large the file, the server may timeout the request (these are for files cumulating larger than a few GB). To prevent this, it is advised you upload portions of your object at a time depending on file size. You can upload exercise solution files, but they will be identified and not distributed through CLARK. A content consumer will be provided your email to obtain solutions directly from you.</p>
 				<div class="picture">
 					<img src="./Resources/CLARK-Materials-Upload.png" width="1000px">
-					<p style="padding-top: 20px;">You have now successfully created your first learning object!</p>
 				</div>
 			</div>
 			<br><br>
-		
+
+			<a name="Children"><div class="section">
+				<h2>Adding Children Learning Objects</h2>
+				<hr>
+				<p>When you upload a large learning object that uses many smaller objects in it, it is useful to add these objects as <i>children</i> to the larger one.  This is to help organize your larger learning objects and allow downloaders to easily find all the parts of your curriculum.</p>
+				<p>To start adding child objects go to the learning object builder, click on the <i>Add/Delete</i> button for Children, and click the option that says <i>Add Child +</i> . From here you have the option to search and choose which learning objects are the children of that parent object.  Note that the option to add children is available for all lengths of objects excluding nanomodules.  It is important to only add the direct children of the specific learning object you are working on. For example, the <i>CLARK Tutorial Learning Object</i> is a micromodule that has two children nanomodules. The <i>CLARK Tutorial Learning Object</i> micromodule could then be the child of a learning object that is either a module or a longer length object.</p>
```

Comment:
We should probably make this more specific based on if you have an existing learning-object you're editing or a new one you are creating.